src/jsonTraverse.ts

```diff
@@ -357,7 +383,7 @@ function parseCommonTypeInfo(typeInfo: ParameterType, type: string, seperator: s
         } else if (item.value) {
             return `"${item.value}"`;
         } else {
-            return 'Object';
+            return parseUserDefinedType(item);
         }
     }).join(seperator);
```

Comment:
type = child.name + ':' + `"${child.type.value}"`;
->

type = `${child.name}: ${child.type.value}`;

Reply:
try to use `` as much as possible + string concat is very hard to read.